.circleci/config.yml

```diff
@@ -185,11 +185,11 @@ commands:
 workflows:
   circleci:
     jobs:
-      - python-3-6: # Oldest supported Python minor version.
+      - python-3-7: # Oldest supported Python minor version.
```

Comment:
Can you change this to `python-earliest` - and the other Python job to `python-latest`? (The goal is to not have to modify Github actions every time we bump a Python release, which means having CircleCI job names that stay consistent.)